src/components/createAppointment/doctorPicker/doctorPicker.js

```diff
@@ -0,0 +1,95 @@
+import { useState, useEffect } from 'react'
+import { useSelector } from 'react-redux'
+import userSelector from '../../../redux/user/userSelector'
+import { Text, PickerWrapper } from '../createAppointment.styles'
+import { Input } from '../../form/styles'
+import { Label, StyledSelect } from './doctorPicker.styles'
+import doctorsList from '../../mainPatient/db.json'
+
+const DoctorPicker = ({ onChangeFormData }) => {
+  const [occupation, setOccupation] = useState('')
+  const [doctorName, setDoctorName] = useState('')
+  const [doctor, setDoctor] = useState({
+    name: '',
+    status: '',
+    description: '',
+    note: '',
+  })
+
+  const options = (elem) =>
+    doctorsList.map((item) => ({
+      value: item[elem],
+      label: item[elem],
+    }))
+
+  const handleChange = (e) => {
+    setDoctor((prev) => ({ ...prev, [e.target.name]: e.target.value }))
+  }
+
+  useEffect(() => {
+    if (!doctorName && !occupation) return
+
+    const data = {
+      name: doctorName.value,
+      status: occupation.value,
+      description: doctor.description,
+      note: doctor.note,
+    }
+
+    onChangeFormData(data, 'formData')
+  }, [doctorName, occupation, doctor])
+
+  return (
+    <PickerWrapper doctorPicker>
+      <Text before third>
```

Comment:
What is third?

Reply:
This is a special prop for display proper `:before` svg for background-image